src/main/java/uk/gov/hmcts/reform/sscs/factory/CcdNotificationWrapper.java

```diff
@@ -196,20 +199,39 @@ && isNotificationEventValidToSendToOtherPartySubscription(otherParty.getOtherPar
         return otherPartySubscription;
     }
 
+    private boolean canSendBasedOnConfidentiality(String partyMember) {
+        if (!(DIRECTION_ISSUED.equals(getNotificationType()) || DIRECTION_ISSUED_WELSH.equals(getNotificationType()))) {
```

Comment:
```suggestion
NotificationEventType notificationType = getNotificationType();
if (!(DIRECTION_ISSUED.equals(notificationType) || DIRECTION_ISSUED_WELSH.equals(notificationType))) {
```

Reply:
You could also refactor and pass this notificationType into other methods looking at rest of class/changes, so we're not calling this get method unnecessarily.